LogAndStream/shimmer3_common_source/Bluetooth_SD/RN4X.c

```diff
@@ -1695,6 +1702,11 @@ uint8_t isBtModuleOverflowPinHigh(void)
 
 void sendNextChar(void)
 {
+    if(sendNextChar_cb)
+    {
+        sendNextChar_cb();
+    }
```

Comment:
consumes 7 CPU cycles when speed test is not in use:
![image](https://github.com/ShimmerResearch/shimmer3/assets/6526073/74092522-6339-49d6-a223-64c3bc6b41f6)